services/notifications/pkg/config/parser/parse.go

```diff
@@ -33,5 +35,22 @@ func ParseConfig(cfg *config.Config) error {
 }
 
 func Validate(cfg *config.Config) error {
+	logger := logging.Configure(cfg.Service.Name, cfg.Log)
+
+	if cfg.Notifications.SMTP.Host != "" {
+		switch cfg.Notifications.SMTP.Encryption {
+		case "tls":
+			logger.Warn().Msg("The smtp_encryption value 'tls' is deprecated. Please use the value 'starttls' instead.")
+		case "ssl":
+			logger.Warn().Msg("The smtp_encryption value 'ssl' is deprecated. Please use the value 'ssltls' instead.")
+		case "starttls", "ssltls":
+			break
+		default:
+			return fmt.Errorf(
+				"Unknown value '%s' for 'smtp_encryption' in service %s. Allowed values are 'starttls', 'ssltls', 'tls' (deprecated) or 'ssl' (deprecated).",
```

Comment:
If the current value set by the user is wrong anyway and the user has to change it, do we need to advertise the deprecated options?

Reply:
@xoxys Not quite sure I understand your comment. What the code is supposed to:

- If the value is set to `tls` or `ssl`, warn the user, but allow to start
- If the value is `starttls` or `ssltls` run normally
- Treat any other value as an error and refuse to start.

Anything missing?

Reply:
The error message says `Allowed values are 'starttls', 'ssltls', 'tls' (deprecated) or 'ssl' (deprecated)."` as the user has to fix its config anyway at this point, why do we need to tell them that `'tls' (deprecated) or 'ssl' (deprecated)` would be a valid option instead of `Allowed values are 'starttls', 'ssltls' and 'none'`

Reply:
Ok. Fixed.